usr/share/rear/restore/NETFS/default/51_set_capabilities.sh

```diff
@@ -0,0 +1,11 @@
+# restore capabilities if capabilities are in the backup
+	if test -s $VAR_DIR/recovery/capabilities ; then
+		Log "Restoring Capabilities."
+		if which setcap >/dev/null 2>&1 ; then
+			while read file cap ; do
+				setcap $cap ${TARGET_FS_ROOT}/${file}	
+			done < <(cat $VAR_DIR/recovery/capabilities | sed 's/=//')
```

Comment:
Maybe something like `while read file equal cap` will avoid the sed. The equal sign will go to the `$equal` variable which you can then simply ignore.